sql/combine.sql

```diff
@@ -366,5 +366,6 @@ FROM(
 WHERE record_id NOT IN (
     SELECT record_id FROM corrections_main
     WHERE field = 'remove'
-)
-;
\ No newline at end of file
+);
+
+CREATE INDEX combined_geom_idx ON combined USING GIST(geom);
```

Comment:
just curious, is this to speed up subsequent uses of of the table `combined`?

Reply:
Yup, there's a fairly long-running join later on